WcaOnRails/config/locales/en.yml

```diff
@@ -995,6 +995,7 @@ en:
       national_records: "History of National Records"
   posts:
     confirm_delete_post: "Are you sure you want to delete this post? This action is not reversible."
+    announcement_info: "Announced by %{author_name} on %{date_time}"
```

Comment:
Could you include a screenshot of what this looks like now? Does the timezone show up in the rendered date?

Currently we display the post times on the homepage in the users's local time. It looks like that will change for the worse with this PR. It would be nice if we came up with a solution that continues to display the time in the user's timezone.